Source/OxyPlot.Maui.Skia/Platforms/iOS/Effects/PlatformTouchEffect.cs

```diff
@@ -88,13 +88,15 @@ public override void TouchesMoved(NSSet touches, UIEvent evt)
     {
         base.TouchesMoved(touches, evt);
 
-        foreach (UITouch touch in touches.Cast<UITouch>())
+        var uitouches = touches.Cast<UITouch>().ToList();
+        foreach (UITouch touch in uitouches)
         {
             long id = ((IntPtr)touch.Handle).ToInt64();
             CheckForBoundaryHop(touch);
             if (idToTouchDictionary[id] != null)
             {
-                FireEvent(idToTouchDictionary[id], id, TouchActionType.Moved, touch, true);
+                //FireEvent with all the points within the touch event
+                FireEvent(idToTouchDictionary[id], id, TouchActionType.Moved, touch, true, uitouches.IndexOf(touch) == uitouches.Count - 1);
```

Comment:
I guess the same multi-point handling that you're using here should also be applied to the `FireEvent` calls in `TouchesBegan` and `TouchesEnded` ?